.changeset/cold-worlds-notice.md

```diff
@@ -0,0 +1,9 @@
+---
+"@hashicorp/design-system-components": major
+---
+
+<!-- START components/dropdown -->
+
+`Dropdown` - Removed the deprecated `@text` argument from the `HdsDropdownListItemInteractive` component.
```

Comment:
Not sure if there are still instances of @text in our consumers codebases, but if there are, in the past we tried to provide brief guidance on how to migrate away from deprecated arguments (you can check the components changelog).

For example
```md
To migrate run the codemod `v4/dropdown-list-item-interactive` (see [readme file](https://github.com/hashicorp/design-system/tree/main/packages/codemods/transforms/v4/dropdown-list-item-interactive))`
```